README.md

```diff
@@ -416,6 +416,19 @@ Note that not every serial program supports the unusual ESP8266 74880bps "boot l
 
 Running `esptool.py --trace` will dump all serial interactions to the standard output (this is *a lot* of output). This can be helpful when debugging issues with the serial connection, or when providing information for bug reports.
 
+## Using esptool.py from Python
+
+esptool.py can easily be integrated into Python applications or called from other Python scripts.
+
+While it currently does have a poor Python API, something which [#208](https://github.com/espressif/esptool/issues/208) will address, it allows for passing CLI 
+arguments to `esptool.main()`. This workaround makes integration very straightforward as you can pass exactly the 
+same arguments as you would on the CLI.
+
+```python
+command = '--baud 460800 read_flash 0 0x200000 flash_contents.bin'
+esptool.main(command.split(' '))
```

Comment:
`split(' ')` will work until there's a space in one of the pathnames :)

What do you think about `command = [ '--baud', '460800', 'read_flash', '0', '0x200000', 'flash_contents.bin']`

It's less readable but it doesn't contain any traps for the unsuspecting. :)

Reply:
> `split(' ')` will work until there's a space in one of the pathnames :)

Argghh, you're right, thanks. I hardly ever think of that because using spaces feels like such a weird thing to do 😉